src/main/java/uk/gov/hmcts/reform/civil/service/AuthorisationService.java

```diff
@@ -0,0 +1,71 @@
+package uk.gov.hmcts.reform.civil.service;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.authorisation.ServiceAuthorisationApi;
+import uk.gov.hmcts.reform.idam.client.IdamClient;
+import uk.gov.hmcts.reform.idam.client.models.UserInfo;
+
+import java.util.Arrays;
+
+@Service
+@Slf4j
+@RequiredArgsConstructor(onConstructor = @__(@Autowired))
+public class AuthorisationService {
+
+    private final ServiceAuthorisationApi serviceAuthorisationApi;
+
+    @Value("${civil.authorised-services}")
+    private String s2sAuthorisedServices;
```

Comment:
this can be straight away be List<String>

Reply:
Done, thanks